builtin/providers/openstack/resource_openstack_lb_loadbalancer_v2.go

```diff
@@ -126,6 +134,25 @@ func resourceLoadBalancerV2Create(d *schema.ResourceData, meta interface{}) erro
 		return err
 	}
 
+	// Once the loadbalancer has been created, apply any requested security groups
+	// to the port that was created behind the scenes.
+	if lb.VipPortID != "" {
```

Comment:
Is it worth letting the Create func call the Update func here? That way you wouldn't need to duplicate the code too much

Will that introduce any more risk?

P.